packages/eslint-config-sentry-app-strict/index.js

```diff
@@ -7,5 +6,18 @@ module.exports = {
   rules: {
     "no-console": ['error'],
     "no-debugger": ['error'],
+
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-is-mounted.md
+    'react/no-is-mounted': ['error'],
+
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-find-dom-node.md
+    // Recommended to use callback refs instead
+    'react/no-find-dom-node': ['error'],
```

Comment:
Sounds like a good idea, but we have a bunch of these that we'd need to update.

Reply:
It will only be in the pre-commit hook (though that does kind of imply if you touch a file with them, you have to fix them, not sure how I feel about that)

Reply:
I think that implication is desirable, it can urge people to update old code that otherwise probably won't be. If they are bulk updating, the hook can always be skipped.